src/main/flight/imu.c

```diff
@@ -634,3 +634,12 @@ bool isUpright(void)
     return true;
 #endif
 }
+
+bool isUpsidedown(void)
+{
+#ifdef USE_ACC
+    return !sensors(SENSOR_ACC) || (attitudeIsEstablished && getCosTiltAngle() < 0);
```

Comment:
This does not look correct logically.
If ACC sensor is not in use, this will report true = upsidedown all the time.
I think it should report false in that case.